advanced/metrics.yaml

```diff
@@ -0,0 +1,187 @@
+          type: array
+          items:
+            type: string
+            enum:
+            - link_down
+            - link_up
+            - bgpv4_route_withdraw
+            - bgpv4_route_advertise
+            - bgpv6_route_withdraw
+            - bgpv6_route_advertise
+            - flow_rx_rate_above_threshold
+            - flow_rx_rate_below_threshold
+
+    AdvancedMetrics.Latency.Request:
+      type: object
+      properties:
+        flow_names:
+          type: array
+          items:
+            type: string
+            x-constraint:
+            - '/components/schemas/Flow/properties/name'
+
+    AdvancedMetrics.Convergence.Request:
+      type: object
+      properties:
+        flow_names:
+          type: array
+          items:
+            type: string
+            x-constraint:
+            - '/components/schemas/Flow/properties/name'
+            example: ['Tx Port -> Rx Port']
+        event_sources:
+          type: array
+          items:
+            type: string
+            x-constraint:
+            - '/components/schemas/Device.BgpRoute/properties/name'
+            example: ['BGP Route Range 1']
+
+    AdvancedMetrics.Response:
+      type: object
+      properties:
+        event_metrics:
+          type: array
+          items:
+            $ref: './event.yaml#/components/schemas/AdvancedMetrics.Event'
+        latency_metrics:
+          type: array
+          items:
+            $ref: '#/components/schemas/AdvancedMetrics.Latency'
+        convergence_metrics:
+          type: array
+          items:
+            $ref: '#/components/schemas/AdvancedMetrics.Convergence'
+
+    AdvancedMetrics.Latency:
+      description: |-
+        The container for latency metrics. The min/max/avg values are dependent 
+        on the type of latency measurement mode that is configured.
+      type: object
+      properties:
+        flow_name:
+          description: |-
+            The name of the source.
+            Flow + metric group?
+          type: string
+          example: 'Tx Port -> Rx Port'
+        minimum_ns:
+          description: |-
+            Minimum latency in nanoseconds
+          type: number
+          format: double
+        maximum_ns:
+          description: |-
+            Maximum latency in nanoseconds
+          type: number
+          format: double
+        average_ns:
+          description: |-
+            Average latency in nanoseconds
+          type: number
+          format: double
+
+    AdvancedMetrics.Convergence:
+      description: |-
+        The container for data plane and control plane convergence metrics
+      type: object
+      properties:
+        flow_name:
+          description: |-
+            The name of the flow that is affected by the trigger event.
+          type: string
+          x-constraint:
+          - '/components/schemas/Flow/properties/name'
+          example: 'Tx Port -> Rx Port'
+        event_source_name:
```

Comment:
Do we need to have event-related information here ?
- There are actually two events that we're internally relying on (above/below threshold and advertise/withdraw routes)
- These events should be available in event metrics

If we need to keep it then I suggest changing the descriptions to remove `BGP` since it is applicable to other protocols too.